db/merge_context.h

```diff
@@ -68,21 +68,21 @@ class MergeContext {
   }
 
   // Get the operand at the index.
-  Slice GetOperand(int index) {
+  Slice GetOperand(int index) const {
     assert(operand_list_);
 
     SetDirectionForward();
     return (*operand_list_)[index];
   }
 
   // Same as GetOperandsDirectionForward
-  const std::vector<Slice>& GetOperands() {
+  const std::vector<Slice>& GetOperands() const {
     return GetOperandsDirectionForward();
   }
 
   // Return all the operands in the order as they were merged (passed to
   // FullMerge or FullMergeV2)
-  const std::vector<Slice>& GetOperandsDirectionForward() {
+  const std::vector<Slice>& GetOperandsDirectionForward() const {
```

Comment:
I think it's worth mentioning here and below that the returned vector is only good until another call to this MergeContext, including potentially a 'const' call.

Reply:
Fixed.